lib/tasks/config.rake

```diff
@@ -38,6 +38,10 @@ namespace :config do
         configuration[:printers] = {
           plate_a: 'g316bc',
           plate_b: 'g311bc2',
+          plate_xp2: 'aa209bc2',
+          ultima_lib_plate: 'aa212bc',
+          ultima_balanced_tube: 'aa209bc1',
+          ultima_balanced_norm_tube: 'ssrtubebc-sq1',
```

Comment:
This is setting a precedent - we don't normally set default printers at a plate purpose level. If we did, then this list would be a lot longer! 

It's a pretty neat solution though, so I don't feel strongly about it.